examples/mission.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import asyncio
+
+from dronecode_sdk import connect as dronecode_sdk_connect
+from dronecode_sdk import (MissionItem, MissionItems)
+
+drone = dronecode_sdk_connect(host="127.0.0.1")
+
+
+async def run():
+    my_items = []
+    my_items.append(MissionItem(47.398039859999997, 8.5455725400000002, 25, 10, True, 0, 0, MissionItem.CameraAction.NONE, 0))
+    my_items.append(MissionItem(47.398036222362471, 8.5450146439425509, 25, 10, True, 0, 0, MissionItem.CameraAction.NONE, 0))
+    my_items.append(MissionItem(47.397825620791885, 8.5450092830163271, 25, 10, True, 0, 0, MissionItem.CameraAction.NONE, 0))
+
+    mission_items = MissionItems(my_items)
+    upload_mission_status, _ = await drone.mission.uploadMission(mission_items)
+    print(f"-- Upload mission: {upload_mission_status}")
+
+    await drone.mission.setReturnToLaunchAfterMission(True)
+
+    arm_status, _ = await drone.action.arm()
+    print(f"-- Arm result: {arm_status}")
+
+    start_mission_status, _ = await drone.mission.startMission()
+    print(f"-- Start mission: {start_mission_status}")
+
+    await drone.mission.setCurrentMissionItemIndex(1)
```

Comment:
So you're skipping the first one on purpose?
That's not that intuitive.

Reply:
Right, I guess I was testing it, but it doesn't have to be in the example.

Reply:
Or it can be but then it's better with a comment about why you're doing it.